meta-mender-nxp/recipes-bsp/u-boot/u-boot-edm_%.bbappend

```diff
@@ -0,0 +1 @@
+require u-boot-mender-edm.inc
```

Comment:
Where does `u-boot-edm` come from? A quick did not give me anything in `meta-freescale/meta-freescale-3rdparty`

Reply:
https://github.com/TechNexion/meta-edm-bsp-release 

I suppose it can be moved into a layer-specific bbappend file. I have not tried this board with either u-boot-fslc or u-boot-imx since the Compulab BSP instructions use u-boot-edm.

Reply:
> https://github.com/TechNexion/meta-edm-bsp-release

But you are not cloning the above layer in the manifest files?  Meaning that this will be incomplete?

Reply:
Updated with proper instructions for using the technexion BSP here: https://hub.mender.io/t/technexion-pico-imx7d-som/451

Reply:
I did a quick test and the instructions do not include Mender and just build the "normal" BSP", I guess you are missing a copy/append of the templates.

Reply:
Updated. And my local tests have my fork added after the repo sync but yes, this won't work unmodified until it gets merged. Bootstrapping, amirite?